src/wp-includes/class-wp-block-bindings-source.php

```diff
@@ -73,16 +72,18 @@ public function __construct( string $name, array $source_properties ) {
 	/**
 	 * Retrieves the value from the source.
 	 *
-	 * @since 6.5.0
```

Comment:
The `@since` annotation should stay here. In addtion, we should add one for `@since 6.7.0 block_bindings_source_value filter was added.`

Reply:
Done in https://github.com/WordPress/wordpress-develop/pull/6839/commits/c4d9b8622f1ac2edb6770abcdced49fd3d2001f9 as test to check that I can push to the contributor's branch.


I'll add. the new 6.7 in a different commit.